.github/workflows/compatibility.yml

```diff
@@ -4,15 +4,16 @@ on:
   pull_request
 
 env:
-  WC_L2_VERSION: '7.3.0'
-  WP_L2_VERSION: '5.9'
+  WC_MIN_VERSION: '7.3.0'
```

Comment:
Should we leave a comment here explaining that this is `L-2` for now? The previous variable names made it obvious.

Reply:
Um, I believe we could just rename it to something like `WC_MIN_SUPPORTED_VERSION` and that could be a little more explicit?

Reply:
Yeah that works better, done!